libs/features/georoutingv2/georoutingv2.js

```diff
@@ -293,10 +308,23 @@ export default async function loadGeoRouting(conf, createTagFunc, getMetadataFun
   if (storedLocale || storedLocale === '') {
     // Show modal when url and cookie disagree
     if (urlLocale.split('_')[0] !== storedLocale.split('_')[0]) {
-      const localeMatches = json.georouting.data.filter((d) => d.prefix === storedLocale);
-      const details = await getDetails(urlGeoData, localeMatches, json.geos.data);
+      const localeMatches = json.georouting.data.filter(
+        (d) => d.prefix === storedLocale,
+      );
+      const details = await getDetails(
+        urlGeoData,
+        localeMatches,
+        json.geos.data,
+      );
       if (details) {
         await showModal(details);
+        sendAnalytics(
+          new Event(
+            `Load:${urlLocale.split('_')[0]}-${
+              storedLocale.split('_')[0]
```

Comment:
You're using `urlLocale.split('_')[0]` and `storedLocale.split('_')[0]` in the conditional on L310 too, maybe these could become variables.

Reply:
done